tests/lcs/core/engine/_ungroupandquit.livecodescript

```diff
@@ -0,0 +1,8 @@
+script "_ungroupandquit"
+on startup
+    create button "one"
+    create button "two"
+    group button "one" and button "two"
+    ungroup the last group
```

Comment:
@livecodepanos aha! that is the issue. ok.. well now the only issue is the release note doesn't match the issue very well

Reply:
Thanks @montegoulding :)